modules/app-gateway-module/config.tf

```diff
@@ -0,0 +1,47 @@
+locals {
+  vnet_rg   = "aks-infra-${var.env}-rg"
+  vnet_name = "core-${var.env}-vnet"
+
+  criticality = {
+    sbox     = "Low"
+    aat      = "High"
+    prod     = "High"
+    ithc     = "Medium"
+    perftest = "Medium"
+  }
+
+  env_display_names = {
```

Comment:
I am not sure if we want to do that.
Ideally, the environment name should be passed from the component which is running the module for a particular environment. With this, we will have to change the module if we want to use this for some other environment.
Plus we already have an environment variable defined and is being passed from the tfvars file for each component run for a particular environment. So why do we want to create another variable.

Reply:
take another look now please